.github/ci-test-connections.yaml

```diff
@@ -139,3 +139,4 @@ connections:
       aws_access_key_id: "ROOTNAME"
       aws_secret_access_key: "CHANGEME123"
       endpoint_url: "http://127.0.0.1:9000"
+      minio: True
```

Comment:
@sunank200 is this a convention we are creating or is this an standard in Airflow? It may be worth to document it, if the first

Reply:
Yes @tatiana we are introducing a parameter `minio=True/False` which user can pass as part of connection.

Reply:
Added documentation here: https://github.com/astronomer/astro-sdk/pull/1767/commits/6646843d1866ef796fbccfab3559b8bda71fae7a